datasets/clinc_oos/README.md

```diff
@@ -52,34 +52,44 @@ pretty_name: CLINC150
 - **Homepage:** [Github](https://github.com/clinc/oos-eval/)
 - **Repository:** [Github](https://github.com/clinc/oos-eval/)
 - **Paper:** [Aclweb](https://www.aclweb.org/anthology/D19-1131)
-- **Leaderboard:**
+- **Leaderboard:** [PapersWithCode](https://paperswithcode.com/sota/text-classification-on-clinc-oos)
 - **Point of Contact:**
 
 ### Dataset Summary
 
-[More Information Needed]
+Task-oriented dialog systems need to know when a query falls outside their range of supported intents, but current text classification corpora only define label sets that cover every example. We introduce a new dataset that includes queries that are out-of-scope (OOS), i.e., queries that do not fall into any of the system's supported intents. This poses a new challenge because models cannot assume that every query at inference time belongs to a system-supported intent class. Our dataset also covers 150 intent classes over 10 domains, capturing the breadth that a production task-oriented agent must handle. It offers a way of more rigorously and realistically benchmarking text classification in task-driven dialog systems.
 
 ### Supported Tasks and Leaderboards
 
-[More Information Needed]
+- `intent-classification`: This dataset is for evaluating the performance of intent classification systems in the presence of "out-of-scope" queries, i.e., queries that do not fall into any of the system-supported intent classes. The dataset includes both in-scope and out-of-scope data. [here](https://paperswithcode.com/sota/text-classification-on-clinc-oos).
 
 ### Languages
 
-[More Information Needed]
+English
 
 ## Dataset Structure
 
 ### Data Instances
 
-[More Information Needed]
+A sample from the training set is provided below:
+```
+{
+    'text' : 'can you walk me through setting up direct deposits to my bank of internet savings account',
+    'label' : 108 
+}
+```
 
 ### Data Fields
 
-[More Information Needed]
+- text : Textual data
+- label : Corresponding to the 150 intent classes over 10 domains
```

Comment:
It would be nice to also include the label list here, to have the label id <-> label name mapping available. What do you think ?

You can find the list here: https://github.com/huggingface/datasets/blob/13d10bb35665370f6096ed47cd0b028b37ba19c0/datasets/clinc_oos/clinc_oos.py#L96

Reply:
That sounds great. Should I add it in a Markdown Table format?
Like the sample below?
| **Label Id** 	| **Label name** 	|
|---	|---	|
| 0 	| restaurant_reviews 	|
| 1 	| nutrition_info 	|
| 2 	| account_blocked 	|
| 3 	| oil_change_how 	|